app/web/features/communities/discussions/DiscussionCard.tsx

```diff
@@ -84,11 +90,19 @@ export default function DiscussionCard({
       <Link href={routeToDiscussion(discussion.discussionId, discussion.slug)}>
         <a>
           <CardContent className={classes.cardContent}>
-            <Avatar
-              user={creator}
-              className={classes.avatar}
-              isProfileLink={false}
-            />
+            {/* Wrapping Avatar and FlagButton in a container for alignment */}
+            <div className={classes.avatarFlagContainer}>
+              <Avatar
+                user={creator}
+                className={classes.avatar}
+                isProfileLink={false}
+              />
+              {/* Adding the FlagButton below the avatar */}
+              <FlagButton
+                contentRef={`discussion/${discussion.discussionId}`}
```

Comment:
Please make this content ref `community/{community id}/discussion/{discussion id}`